lite/examples/object_detection/android/README.md

```diff
@@ -46,11 +46,11 @@ git clone https://github.com/tensorflow/models/
 ```
 * Build and install this repository.
 ```
-cd models
+cd models/research
 python3 setup.py build && python3 setup.py install
 ```
 * Download the MobileNet SSD trained on **[Open Images v4](https://storage.googleapis.com/openimages/web/factsfigures_v4.html)** **[here](https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/detection_model_zoo.md)**. Extract the pretrained TensorFlow model files.
-* Go to `models/research` directory and execute this code to get the frozen TensorFlow Lite graph.
+* Within `models/research` directory execute this code to get the frozen TensorFlow Lite graph.
```

Comment:
The original sentence sounds more natural to me. Could you revert this line?

Reply:
@yyoon reverted. Thanks